packages/neuron-wallet/src/keys/keystore.ts

```diff
@@ -0,0 +1,16 @@
+export interface Master {
```

Comment:
`Master/Child` is too generic names (since we don't have namespaces to scope them).

Will `MasterKey/ChildKey` be better? (just an idea, we should think of a good name.)

Reply:
I think `Master/Child` are existed in `keys` folder which have showed this module is about `key`, and file name keystore also has the same effect.